Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h

```diff
@@ -73,12 +73,12 @@ class AsyncScrollingCoordinator : public ScrollingCoordinator {
 
     virtual void hasNodeWithAnimatedScrollChanged(bool) { };
     
+    WEBCORE_EXPORT void setUserInterfaceLayoutDirection(ScrollableArea&, UserInterfaceLayoutDirection) override;
```

Comment:
I think this should be called `scrollbarLayoutDirection` everywhere; it's less general that "user interface layout direction".

Reply:
Fixed.